src/js/components/Accordion.js

```diff
@@ -63,7 +70,12 @@ export default class Accordion extends Component {
 
 Accordion.propTypes = {
   animate: PropTypes.bool,
+  disabled: PropTypes.bool,
   openMulti: PropTypes.bool,
+  icon: PropTypes.oneOfType([
+    PropTypes.element,
```

Comment:
I agree, we should keep controls consistent across the application, but what about the other users using grommet on personal or non hpe related projects and would want to use the `next` icon rather than `tab-next`. 

Not that I want to use the `next` icon, just giving an example. 😄